src/components/data/listing/TableView.js

```diff
@@ -87,7 +87,13 @@ function TableView(props) {
     const tableColumns = getTableColumns(isMedium, isLarge);
 
     return (
-        <TableContainer component={Paper}>
+        <TableContainer
+            component={Paper}
+            style={{
+                minHeight: 320,
+                maxHeight: isMedium || isLarge ? 800 : 320,
```

Comment:
I _think_ (haven't tried this) we can get away with doing something like `100vh` (see Relative Lengths [here](https://www.w3schools.com/cssref/css_units.asp)).  The only thing is there might have to be a calculation to subtract the height of the app bar and navigation for it to work properly.  That could get complicated, but I think it might serve us better than having a set height.  (Again, don't know if it'll actually work, but I think it will)